src/styles/transitions.spec.js

```diff
@@ -2,9 +2,25 @@
 /* eslint-env mocha */
 
 import { assert } from 'chai';
+import { spy } from 'sinon';
 import transitions, { easing, duration } from './transitions';
 
 describe('transitions', () => {
+  let originalConsoleError;
+
+  beforeEach(() => {
+    /* eslint-disable no-console */
+    originalConsoleError = console.error;
```

Comment:
We could be using the `stub` API of *sinon* instead of the spy. I know that @nathanmarks isn't a fan of doing such thing with the console. But I think that it worth it.